packages/clay-tabs/src/Item.tsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+import {ElementType} from './types';
+
+interface IProps extends Omit<React.HTMLAttributes<HTMLLIElement>, 'onClick'> {
+	/**
+	 * Flag to indicate if the component is active or not.
+	 */
+	active?: boolean;
+
+	/**
+	 * Tab Item component to render. Can be an 'anchor' or a 'button'.
+	 */
+	component?: ElementType;
+
+	/**
+	 * Flag to indicate if the TabPane is disabled.
+	 */
+	disabled?: boolean;
+
+	/**
+	 * Flag to indicate if the Tab Item is a dropdown.
+	 */
+	dropdown?: boolean;
+
+	forwardRef?: React.Ref<any>;
+
+	/**
+	 * Props to be added to the item element that can be an anchor or a button.
+	 */
+	itemElementProps?: React.HTMLAttributes<
+		HTMLAnchorElement | HTMLButtonElement
+	>;
+
+	/**
+	 * Registers the `tabkey` for the Tab Item.
+	 */
+	tabkey: number;
+
+	/**
+	 * Callback to be used when clicking to a Tab Item.
+	 */
+	onClick?: (val: number) => void;
+
+	/**
+	 * The path to the SVG spritemap file containing the icons.
+	 */
+	spritemap?: string;
+
+	/**
+	 * Name of the Tab
+	 */
+	tabName: string;
+}
+
+const TabHeader = React.forwardRef(
+	(
+		{
+			active = false,
+			component = 'button',
+			disabled = false,
+			dropdown = false,
+			forwardRef,
+			itemElementProps = {},
+			tabkey,
+			onClick,
+			spritemap,
+			tabName,
+		}: IProps,
+		IProps
+	) => {
+		if (component === 'anchor') {
+			return (
+				<>
+					<a
```

Comment:
Could we use `ClayLink` here?

Reply:
As discussed internally, we're waiting for a component to resolve when to use Button and when use Anchor. This code review is outdated. Currently is using an `ElementTag` like implementation.